### PR TITLE
Feature/tt 1891 2

### PR DESCRIPTION
**Pull request by sarahentzel**

### CODE REVIEW

File: localization/captureFallback-new.xlf [hunk 1]
```diff
@@ -2443,18 +2443,6 @@
             <target>Sim</target>
          </segment>
       </unit>
-      <unit id="transcriber.aheadTip">
```
Comment (gtryus) [inline localization/captureFallback-new.xlf:2446]: I am not sure how this file is used? was it just a temporary file?

File: src/components/MediaTab.tsx [hunk 2]
```diff
@@ -176,9 +176,9 @@ interface IRecordProps {
 
 interface IProps
   extends IStateProps,
-    IDispatchProps,
-    IRecordProps,
-    WithDataProps {
+  IDispatchProps,
```
Comment (gtryus) [inline src/components/MediaTab.tsx:179]: why are these formatting differently on your computer and on mine?

File: src/components/PassageRecord.tsx [hunk 3]
```diff
@@ -217,9 +218,9 @@ function PassageRecord(props: IProps) {
           onClick={handleAddOrSave}
           variant="contained"
           color="primary"
-          disabled={(ready && !ready()) || !filechanged || name === ''}
+          disabled={(ready && !ready()) || name === ''} //|| !filechanged}
```
Comment (gtryus) [inline src/components/PassageRecord.tsx:221]: should this comment just be removed?

File: src/crud/useWaveSurfer.tsx [hunk 4]
```diff
@@ -87,27 +91,14 @@ export function useWaveSurfer(
...
-      'hasregion?',
-      regionRef.current !== undefined,
-      'looping?',
-      regionRef.current?.loop
-    );
     playingRef.current = playing;
     if (playingRef.current) {
       if (wsRef.current?.isReady) {
-        if (regionRef.current && regionRef.current.loop)
+        if (regionRef.current && regionRef.current.loop)//this still needs work
```
Comment (gtryus) [inline src/crud/useWaveSurfer.tsx:97]: We should probably use "TODO:" at the beginning of comments like this so we can easily search for them.

**Review by gtryus — CHANGES_REQUESTED** [review]
Look over the comments and see what you think.

File: localization/captureFallback-new.xlf (hunk 1 above)
Comment (sarahentzel) [reply localization/captureFallback-new.xlf:2446]: I have no idea. There are so many localization files now, and directories and I'm not sure what needs to be updated.  When I delete the az, hi, nl, tl directories, they come back.  It makes it harder to find which localization file I want.

File: src/components/MediaTab.tsx (hunk 2 above)
Comment (sarahentzel) [reply src/components/MediaTab.tsx:179]: sigh...we'll have to zoom to figure it out...

File: src/components/PassageRecord.tsx (hunk 3 above)
Comment (sarahentzel) [reply src/components/PassageRecord.tsx:221]: I really didn't want to pick through what I was still working on to get the fix for Michael checked in.  I'm still working on verifying that saving with different file types actually does something.

File: localization/captureFallback-new.xlf (hunk 1 above)
Comment (sarahentzel) [reply localization/captureFallback-new.xlf:2446]: I removed this file

**Review by gtryus — APPROVED** [review]

File: src/components/MediaTab.tsx (hunk 2 above)
Comment (sarahentzel) [reply src/components/MediaTab.tsx:179]: I didn't have all the extensions installed on my new local user.  I have them now, so hopefully this won't continue.

**Pull request merged**